src/index.css

```
@@ -69,3 +65,82 @@ button:focus-visible {
     background-color: #F9F9F9;
   }
 }
+
+h2 {
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 43px;
+  color: #FFF;
+}
+
+@media (max-width: 736px) {
+
+  h2 {
+    font-size: 19px;
+    line-height: 32px;
+  }
+}
+
+p {
+  font-size: 22px;
+  color: #FFF;
+  font-weight: 700;
+  line-height: 32px;
+}
+
+p {
+  font-size: 22px;
+  color: #FFF;
+  font-weight: 700;
+  line-height: 32px;
+}
```

🙃 I think, you can remove duplications and describe only one entity of each CSS [selector](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Selectors)( class or tag). For whole this block `@media (max-width: 736px) {...}`
```suggestion
p {
  font-size: 22px;
  color: #FFF;
  font-weight: 700;
  line-height: 32px;
}

```

---

👍